typings.json

```diff
@@ -1,11 +1,11 @@
 {
-  "ambientDependencies": {
+  "globalDependencies": {
+    "adm-zip": "registry:dt/adm-zip#0.0.0+20160317120654",
     "chalk": "registry:dt/chalk#0.4.0+20160317120654",
     "form-data": "registry:dt/form-data#0.0.0+20160316155526",
-    "glob": "registry:dt/glob#5.0.10+20160317120654",
-    "minimatch": "registry:dt/minimatch#2.0.8+20160317120654",
-    "node": "registry:dt/node#4.0.0+20160412142033",
-    "optimist": "registry:dt/optimist#0.0.0+20160316171810",
+    "jasmine": "registry:dt/jasmine#2.2.0+20160412134438",
+    "minimist": "registry:dt/minimist#1.1.3+20160317120654",
+    "node": "registry:dt/node#4.0.0+20160423143914",
```

Comment:
Add back `glob`, `minimatch`, and `optimist`. Remove `jasmine` and `minimist`. When running `typings install` if the typings dependencies are out of date, please update them.